setup.py

```diff
@@ -1,13 +1,21 @@
 from setuptools import setup
- 
+from setuptools import find_packages
+
 package_name = 'bno055'
- 
+
 setup(
     name=package_name,
-    version='0.0.0',
-    packages=[],
+    version='1.0.0',
```

Comment:
same as above, minor version for now until we reach a more mature phase.

```suggestion
    version='0.1.0',
```

Reply:
done